.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report a bug, false-positive or false-negative
+title: '[BUG] '
+labels: ''
+assignees: ''
```

Comment:
Do you want to have some people assigned automatically?

Reply:
No this is fine.